src/Zicht/Bundle/AdminBundle/Service/Quicklist.php

```diff
@@ -134,7 +134,10 @@ function ($a, $b) use ($pattern) {
             }
         );
 
-        return array_slice($results, 0, $max);
+        $repoConfig = $this->repos[$repository];
+        $maxResults = isset($repoConfig['max_results']) ? $repoConfig['max_results'] : $max;
```

Comment:
Ik zou dan eerder verwachten dat als je een max meegeeft aan deze functie, dat dat de config overschrijft.

Dan zou je moeten doen `$max = null` (en `@param int|null $max`). En dan hier:

```suggestion
        $maxResults = $max !== null ? $max : $repoConfig['max_results'];
```

(Die isset check lijkt me overbodig, omdat de repo config `max_results` node een default heeft)

Reply:
Aha, andersom, ja dat kan natuurlijk ook :smile: